src/Symfony/Component/Yaml/Tests/ParserTest.php

```diff
@@ -66,6 +66,37 @@ public function getDataFormSpecifications()
         return $tests;
     }
 
+    public function testLineBreaksBeforeDocumentSeparatorAndComment()
+    {
+        $yaml = <<<'EOF'
+
+ 
+# Two empty lines above (second - with one whitespace) and two below (first - with three whitespaces).
+   
+
```

Comment:
Yeah, I've already did it. You can take a look:
- https://github.com/symfony/symfony/pull/20660/files#diff-d02f22bcb80a46c92e40ec024bc02995R83
- https://github.com/symfony/symfony/pull/20660/files#diff-d02f22bcb80a46c92e40ec024bc02995R102